examples/client/beatBuilder.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<head>
+    <title>Crunker test</title>
+    <script src="https://unpkg.com/crunker@latest/dist/crunker.js"></script>
+    <script type="text/javascript">
+        mergeOneLine = async function(crunker, buffer, times){
+            let audios = times.map(t => crunker.padAudio(buffer, 0, t))
+            return crunker.mergeAudio(audios)
+        }
+        
+        beatBuilder = async function(crunker, beatConf, buffers, beatTiming){
+            let beatLines = await Promise.all(buffers.map((_,i) => {
+                return mergeOneLine(crunker, buffers[i], beatTiming[i].map(bt => bt*beatConf.timing))
+            }))
+            let beat = await crunker.mergeAudio(beatLines)
+            return await mergeOneLine(crunker, beat, Array.from(Array(beatConf.repeats).keys()).map(e => beatConf.delayStart+e*beatConf.beats*beatConf.timing))
```

Comment:
`prefer`: name this with a `const` that descibes what this is doing `Array.from(Array(beatConf.repeats).keys()` 

Same with this line: `beatConf.delayStart+e*beatConf.beats*beatConf.timing`, can you split it out with spaces and name the variable for example...

e.g. ` const beatTiming = (beatConf.delayStart + e) * (beatConf.beats * beatConf.timing)

This just helps make things easier to read, which is important.